### ISSUE TITLE
Analysis of unrecognized days

### ISSUE THREAD

**Issue by Kirens** [akaProxy/NOVAXML.JS.old#10]
If a day consists of two parallel lessons, the current analysis does not recognize the lessons.
A script is needed that can catch these days and return a `Day object`

**Comment by jdbosser** [comment]
Yes, instead of rendering a whole week incorrectly. There are still fully analyzable days in the week, so if we can find the day that is wierd, atleast we can export 80% of the week correctly :)
